backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/workflow/Workflow_RestController_StepDef.java

```diff
@@ -232,17 +232,21 @@ else if (event.equals("ReceiveFrom"))
 					.wfProcessId(workflowProcess.getAsString())
 					.wfActivityId(workflowActivity.getAsString())
 					.receiveFrom(JsonManufacturingOrderEvent.ReceiveFrom.builder()
-							.lineId(workflowLine.getId())
-							.qtyReceived(workflowLine.getQtyToReceive())
-							.aggregateToLU(JsonLUReceivingTarget.builder()
-									.newLU(JsonNewLUTarget.builder()
-											.luCaption(receivingTargetValues.getLuCaption())
-											.tuCaption(receivingTargetValues.getTuCaption())
-											.luPIItemId(HuPackingInstructionsItemId.ofRepoId(receivingTargetValues.getLuPIItemId()))
-											.tuPIItemProductId(HUPIItemProductId.ofRepoId(receivingTargetValues.getTuPIItemProductId()))
-											.build())
-									.build())
-							.build());
+										 .lineId(workflowLine.getId())
+										 .qtyReceived(workflowLine.getQtyToReceive())
+										 .aggregateToLU(JsonLUReceivingTarget.builder()
+																.newLU(JsonNewLUTarget.builder()
+																			   .luCaption(receivingTargetValues.getLuCaption())
+																			   .tuCaption(receivingTargetValues.getTuCaption())
+																			   .luPIItemId(HuPackingInstructionsItemId.ofRepoId(receivingTargetValues.getLuPIItemId()))
+																			   .tuPIItemProductId(HUPIItemProductId.ofRepoId(receivingTargetValues.getTuPIItemProductId()))
+																			   .build())
+																.build())
+										 .bestBeforeDate(row.getAsOptionalString("BestBeforeDate").orElse(null))
+										 .catchWeight(row.getAsOptionalBigDecimal("CatchWeight").orElse(null))
```

Comment:
consider using 
```java
row.getAsOptionalQuantity("CatchWeight", uomDAO::getByX12DE355)
```

that will allow you writing "0.5 KGM" in CatchWeight column, in feature file.